lib/config.js

```diff
@@ -32,11 +32,23 @@ function expandTilde(apath) {
     return path.join(homedir, apath.slice(2));
```

Comment:
Though you commented on line 32, I think you meant to comment on the resolveEnvVars() function. I've updated the pull request with documentation on that function. 

For more context, for deploying Guerilla on multiple machines it is nice if config files such as used in the node PM2 module can be immutable, and only vary by an environment variable. This update enables that by allowing PM2 (or other launchers) to start Guerilla and allow the environment variables to vary the actual config file.